### PR TITLE
Add links to the sidebar for Static Pages

### LINKED ISSUE #55: Add links to the sidebar for Static Pages
We want to add dynamically in the sidebar the links to static pages generated in `PagesController`.
* You can find the sidebar file in `app/views/shared/_sidebar.html.slim`

If you are not familiar with slim, have a look at [slim-lang](http://slim-lang.com/)

### PR DESCRIPTION
**Pull request by AnkushMalik**
Move the team info link inside a loop

Closes #55

### CODE REVIEW

File: app/views/shared/_sidebar.html.slim [hunk 1]
```diff
@@ -23,8 +23,9 @@
       i.users.icon
       | TEAM
     .content
-      .item
-        = link_to 'Team Info', build_solutions_team_page_path
+      - PagesController::PAGES.each_with_index do |pg,i|
+        .item
+          = link_to "Team info #{i+1}", "/pages/#{pg}"
```
Comment (Ana06) [inline app/views/shared/_sidebar.html.slim]: The name for the link should be more explicit.

You could for example change [the array in PagesController](https://github.com/openSUSE/agile-team-dashboard/blob/master/app/controllers/pages_controller.rb#L2) by a hash where the value is the name. You will also need to update [the line 4 of this file](https://github.com/openSUSE/agile-team-dashboard/blob/master/app/controllers/pages_controller.rb#L4) when doing this.

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 class PagesController < ApplicationController
-  PAGES = [:build_solutions_team].freeze
+  PAGES = [{ path: :build_solutions_team, path_title: 'Team Info' }].freeze
```
Comment (Ana06) [inline app/controllers/pages_controller.rb]: you need to use the action name as key, if not when adding more pages this won't work. So, like this:

``` Ruby
PAGES = [{ build_solutions_team: 'Team Info' }].freeze
```

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
 
   PAGES.each do |action_name|
```
Comment (Ana06) [inline app/controllers/pages_controller.rb]: if you use the action name as key, here you can write `PAGES.each_key do |action_name|` and the rest of the code keep the same 😉

File: app/views/shared/_sidebar.html.slim [hunk 4]
```diff
@@ -23,8 +23,9 @@
       i.users.icon
       | TEAM
     .content
-      .item
-        = link_to 'Team Info', build_solutions_team_page_path
+      - PagesController::PAGES.each do |pg, title|
```
Comment (Ana06) [inline app/views/shared/_sidebar.html.slim]: what about using a more explicit name than `pg`? `path` for example 🤔

Comment (AnkushMalik) [reply app/views/shared/_sidebar.html.slim]: ok, 'path' is better :+1:

**Pull request merged**